.eslintrc.json

```diff
@@ -1,3 +1,47 @@
 {
-  "extends": ["next/core-web-vitals", "prettier"]
+  "extends": ["next/core-web-vitals", "prettier"],
+  "parserOptions": {
+    "sourceType": "module"
```

Comment:
What does this option do?

Reply:
This is saying that EMCAScripts are used in the project.

However, that is not really the case and this option is overkill to have here. I'll remove it.